packages/exoflex/src/components/Switch.tsx

```diff
@@ -36,8 +37,15 @@ export default function Switch(props: SwitchProps) {
   } = props;
   let { colors, style: themeStyle } = useTheme();
   let [xValue] = useState(new Animated.Value(value ? 1 : 0));
+  let width = +(StyleSheet.flatten(trackStyle)?.width || widthProps);
 
-  let thumbSize = width / 2 - 2 * MARGIN;
+  let thumbSize =
+    +(
+      StyleSheet.flatten(themeStyle?.switch?.thumbStyle)?.width ||
+      StyleSheet.flatten(thumbStyle)?.width ||
+      width / 2
+    ) -
+    2 * MARGIN;
```

Comment:
Hmmm, the order of precedence looks weird 🤔 

Right now it's `themeStyle > thumbStyle > width`.
I think it should be `thumbStyle > themeStyle > width`, where the `thumbStyle` gets the priority because this is used only for the `thumbSize`.

What do you think?

Whatever we pick, we probably need to add this to the docs though.

Reply:
Oops, nice catch! Yeah it should be `thumbStyle > themeStyle > width`, I think I pasted it at the wrong position without realizing that it should be the opposite order from the style below.

Reply:
> Whatever we pick, we probably need to add this to the docs though.

Absolutely!